lua/entities/gmod_wire_expression2/core/custom/constraintcore.lua

```diff
@@ -678,3 +741,43 @@ e2function void entity:constraintBreak(string consType, entity ent2)
 		end
 	end
 end
+
+registerCallback( "postinit", function()
+	E2Lib.currentextension = "constraintcore"
+	local forms = { "e:e", "e:b", "b:e", "b:b" }
+	local weldpars = "nn"
+	
+	for k,v in pairs(forms) do
+		for i = 0, 2, 2 do
+			registerFunction("weld",v..weldpars:sub(0,i), "", function(self, args)
+				print(args)
```

Comment:
Looks like some debug prints are still in these

Reply:
Probably would've been cleaner to just write the functions by hand rather than programmatically generating them.

Reply:
Thanks for catching that. Should I restructure it to do that? I wanted to avoid that just so any future modifications wouldn't have to edit each line by hand.

Reply:
I'll leave it up to Vurv. They know e2 better than me